sshcode.go

```diff
@@ -308,6 +314,13 @@ func randomPort() (string, error) {
 // checkSSHDirectory performs sanity and safety checks on sshDirectory, and
 // returns a new value for o.reuseConnection depending on the checks.
 func checkSSHDirectory(sshDirectory string, reuseConnection bool) bool {
+
+	if runtime.GOOS == "windows" {
+		flog.Info("OS is windows, disabling connection reuse feature")
+		//reuseConnection = false
```

Comment:
Unnecessary comment.

Reply:
It is unclear what you mean by this. Should i uncomment it? or just straight up delete it?

Reply:
Just delete the line, should be fine.